projects/core/components/tooltip/tooltip.component.ts

```diff
@@ -65,8 +64,4 @@ export class TuiTooltipComponent extends TuiHintControllerDirective {
             event.stopPropagation();
         }
     }
-
-    get icon(): PolymorpheusContent {
-        return this.options.icon;
```

Comment:
![tooltip](https://user-images.githubusercontent.com/35179038/182810675-9d1b5754-c8b8-4403-893f-fbaae96300c1.png)

Reply:
https://taiga-ui--pr2273-chart-hints-5rshlhio.web.app/components/tooltip#options
vs
https://taiga-ui.dev/components/tooltip#options

Reply:
Also in dark theme

<img width="748" alt="image" src="https://user-images.githubusercontent.com/12021443/182811206-4d0f4b3e-1bb4-4d16-90ba-9cbf2ba23046.png">

Reply:
Fixed :)